tests/test_helpers.nim

```diff
@@ -1,7 +1,9 @@
 import
-  unittest, chronos,
+  unittest, chronos, bearssl,
```

Comment:
Why are we adding bearssl? Is that where the RNG stuff is?

Reply:
Yeah